servers/testkube/server.yaml

```
@@ -0,0 +1,49 @@
+name: testkube
+image: mcp/testkube
+type: server
+meta:
+  category: devops
+  tags:
+    - devops
+    - testing
+    - automation
+    - orchestration
+    - kubernetes
+    - gitops
+    - playwright
+    - cypress
+    - selenium
+    - k6
+    - postman
+    - jmeter
+
+about:
+  title: Testkube MCP Server
```

Currently our validation doesn't allow for `MCP` in the title
```suggestion
  title: Testkube
```